setup.py

```diff
@@ -69,7 +69,7 @@
         'pandas', 'lxml', 'uncertainties'
     ],
     'extras_require': {
-        'test': ['pytest', 'pytest-cov'],
+        'test': ['pytest', 'pytest-cov', 'colorama'],
```

Comment:
I'm ok with the addition of 'colorama'; my rule for being ok: if in the latest Ubuntu LTS' repository, its probably an acceptable dependency (so long as it doesnt affect build complexity, etc.)

Reply:
Yeah, it's a pretty simple one and is really widely used (one of the most popular packages on PyPI).